tap_recharge/client.py

```diff
@@ -187,7 +187,7 @@ def request(self, method, path=None, url=None, **kwargs): # pylint: disable=too-
         # If we did not specify any API Version during API Call, the Recharge will use the default API Version of our store
         # the 'collections' was added as part of API Version: '2021-11', for older API Version,
         # we will get empty records so adding 'X-Recharge-Version' for 'collections' API call
-        if path == 'collections':
+        if path in ['collections', 'store']:
```

Comment:
Upgraded API version only for `Store` as part of this PR. As for some fields, there is schema change in the new version that will result in a Transform error.